tests/sentry/api/endpoints/test_organization_details.py

```diff
@@ -783,6 +785,39 @@ def test_org_mapping_already_taken(self):
         OrganizationMapping.objects.create(organization_id=999, slug="taken", region_name="us")
         self.get_error_response(self.organization.slug, slug="taken", status_code=409)
 
+    def test_configure_auth_provider(self):
+        provider_config = {"option": "old"}
+        updated_provider_config = {"option": "updated"}
```

Comment:
Updated is a little long. I'd usually use `old` and `new`, or `prev`/`curr`/`next`. Having the same number of chars helps the variable names line up with monospace font.

Reply:
Changed to `new_config` and `old_config`